docs/modules/redpanda.md

```diff
@@ -0,0 +1,38 @@
+# Redpanda
+
+Testcontainers can be used to automatically instantiate and manage [Redpanda](https://redpanda.com/) containers.
+More precisely Testcontainers uses the official Docker images for [Redpanda](https://hub.docker.com/r/vectorized/redpanda/)
```

Comment:
this works ok, we added a small translation via scarf.sh that allows us to point to a different backend while giving users aa stable name - this works today but would prevent us from moving to amazon or smth like that. 

just a nit, but would be easier to support via 
```
docker.redpanda.com/vectorized/redpanda:$VERSION 
```